New package: PlotPWM v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: PlotPWM
- Repository: https://github.com/kchu25/PlotPWM.jl
- Created by: @kchu25
- Version: v1.0.0
- Commit: c27f22f8b482eb66df940b3b15eeebb26cee0715
- Reviewed by: @kchu25
- Reference: https://github.com/kchu25/PlotPWM.jl/commit/c27f22f8b482eb66df940b3b15eeebb26cee0715#commitcomment-146004246
- Description: Plot position weight matrices (PWM)
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
" and it will be added to the registry PR, and if TagBot is installed it will also be added to the release that TagBot creates. i.e.
> 
> ```
> @JuliaRegistrator register
> 
> Release notes:
> 
> ## Breaking changes
> 
> - blah
> ```
> 
> To add them here just re-invoke and the PR will be updated.
> 
> ### Tagging
> After the above pull request is merged, it is recommended that a tag is created on this repository for the registered package version.
> 
> This will be done automatically if the [Julia TagBot GitHub Action](https://github.com/marketplace/actions/julia-tagbot) is installed, or can be done manually through the github interface, or via:
> 
> ```
> git tag -a v1.0.0 -m "<description of version>" c27f22f8b482eb66df940b3b15eeebb26cee0715
> git push origin v1.0.0
> ```

[noblock]
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd629ebcc19684da90f8e5e0a8ce76a2e510767b0252e6976200a2ef6351b48c0777f023448a6126a9fb8abf30d64e4331d6f67b1456118e38bd57d77b76f5ab7a622fe89fc2a9a6e9d2d5244b9d05632f07519658397bfc8adb4ed656e1240fdb2fdc4e9d18cee2f14f51012b113a3c5574de15119ee2802b5b98f040b6352bdcb4231b419152e0527278b067be645896fead90f3f7a04a86cf2adf153ed04c89706a1ceb6f55886b5f5606d1161e8c3a0 -->

kchu25 commented:
> Should this be closed in favor of #114017? The name `PlotPWM` does not seem very problematic to me (although some people are more allergic than me to acronyms); this was set to merge automatically in three days.

Hey Goerz thanks for your feedback! I'd rather use this `PlotPWM` as I think this is a more straightforward name. If this is ok then please close #114017 or I can remove that repo.

goerz commented:
[noblock] In order for `PlotPWM` to be merged, you have to edit your https://github.com/JuliaRegistries/General/pull/113990#issuecomment-2318296898 to include `[noblock]`

kchu25 commented:
[noblock]

JuliaRegistrator committed:
New package: PlotPWM v1.0.0

UUID: 044a62f1-e260-4c1b-b41b-507424d69d9a
Repo: https://github.com/kchu25/PlotPWM.jl.git
Tree: fdbe4720a91898aba33bbb646b093d9bc52149fb

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

kchu25 commented:
[noblock]
